problems/2.functions.js

```diff
@@ -20,7 +23,9 @@ function greeting() {
 * @returns {null}
 * @modifies {global.globalCount}
 */
-function incrementGlobalCount() {
+function incrementGlobalCount(num) {
+  let num =  global.globalcount;
+  
```

Comment:
To update a variable you need to do reassignment and thus need the assignment operator (=). 
global.globalCount += val